### PR TITLE
docs: add hugegraph-server docker deployment guide

### PR DESCRIPTION
**Pull request by Radeity**

### CODE REVIEW

File: content/en/docs/quickstart/hugegraph-server.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ There are three ways to deploy HugeGraph-Server components:
 - Method 1: One-click deployment
 - Method 2: Download the binary tarball (most recommended)
 - Method 3: Source code compilation
-- Method 4: Use Docker container (BETA)
```
Comment (imbajin) [inline content/en/docs/quickstart/hugegraph-server.md:34]: THX

**Review by imbajin — APPROVED** [review]
Also add hubble-docker instruction in tasks（assign to others）

**Comment by liuxiaocs7** [comment]
> Also add hubble-docker instruction in tasks（assign to others）

Hi, @imbajin, some features in this issue: https://github.com/apache/incubator-hugegraph/issues/840 can also be add to tasks?

**Review by liuxiaocs7 — APPROVED** [review]

**Comment by imbajin** [comment]
> > Also add hubble-docker instruction in tasks（assign to others）
> 
> Hi, @imbajin, can some features in this issue: https://github.com/apache/incubator-hugegraph/issues/840 also be add to tasks too?
> 
> 

 good reminder，I'll add it soon (and the docker tasks are friendly to the new devs)

**Comment by imbajin** [comment]
> > Also add hubble-docker instruction in tasks（assign to others）
> 
> Hi, @imbajin, can some features in this issue: https://github.com/apache/incubator-hugegraph/issues/840 also be add to tasks too?
> 
> 

 good reminder，I'll add it soon (and the docker tasks are friendly to the new devs)

**Pull request merged**